### ISSUE TITLE
Selenium/chromium is failing to download JSON component file to the indicated target path for download files and therefore test is failing.

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#1178]
@woodt - Please note:

**Describe the bug as a Story**
Selenium/chromium is failing to download JSON component file to the indicated target path for download files and therefore test is failing.

**To Reproduce**
1. Run `python manage.py test controls.tests.ComponentUITests.test_component_download_oscal_json`
1. Watch Selenium run Chromium in real-time and see file downloaded (in my case on MacOS, to my `~/Downloads` directory and not the download directory configured for Chromium.
1. Wait for test to complete.
1. See error.

The following is output of test run with additional debug code added to print out the set download path. File is not downloaded to set download path. Instead, file is downloaded to Chrome's default download directory.

```
(venv) greg@Gregs-MacBook-Pro /codedata/code/govready-q-master (develop) $ python manage.py test controls.tests.ComponentUITests.test_component_download_oscal_json
INFO: ALLOWED_HOSTS ['localhost']
INFO: Connection scheme is 'http'.
INFO: 'SITE_ROOT_URL' set to http://localhost:8000 
WARNING: Specified PDF generator is not supported. Setting generator to 'off'.
WARNING: Specified IMG generator is not supported. Setting generator to 'off'.
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
{'_arguments': ['disable-infobars', '--window-size=1200,1200', '--incognito'],
 '_binary_location': '',
 '_caps': {'browserName': 'chrome',
           'goog:chromeOptions': {'args': ['disable-infobars',
                                           '--window-size=1200,1200',
                                           '--incognito'],
                                  'extensions': []},
           'platform': 'ANY',
           'version': ''},
 '_debugger_address': None,
 '_experimental_options': {'prefs': {'download.default_directory': '/var/folders/rk/cn5lpv2s2g1cwplxcrdxzj1r0000gn/T',
                                     'download.directory_upgrade': True,
                                     'download.prompt_for_download': False,
                                     'safebrowsing.enabled': True}},
 '_extension_files': [],
 '_extensions': []}
********* self.json_download /var/folders/rk/cn5lpv2s2g1cwplxcrdxzj1r0000gn/T/xyzzy.json
2020-12-11 10:20:49,786 siteapp.good_settings_helpers level ERROR login ip=127.0.0.1 username=me result=user:2
2020-12-11 10:20:49,817 siteapp.views level INFO {"user": {"id": 2, "username": "me"}, "event": "project_list"}
F
======================================================================
FAIL: test_component_download_oscal_json (controls.tests.ComponentUITests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/codedata/code/govready-q-master/controls/tests.py", line 237, in test_component_download_oscal_json
    self.assertTrue(self.json_download.is_file())
AssertionError: False is not true

----------------------------------------------------------------------
Ran 1 test in 13.645s

FAILED (failures=1)
Destroying test database for alias 'default'...
```

**Expected behavior**
Test should pass because test scripts configure Selenium/Chromium to download to a particular directory. But file is not downloaded to that directory.

**Desktop (please complete the following information):**
- Branch `develop` as of 12/11/2020
 - OS: MacOS 14.4
 - Selenium/Chromium
 - Whatever is set by test scripts. Local computer Chrome is Version 87.0.4280.88 (Official Build) (x86_64)

**Comment by woodt** [comment]
It looks like there was a chunk ordering issue when this work was merged.  The download webdriver options are being set *after* the webdriver object is created.

**Comment by gregelin** [comment]
Fixed in commit e58a0d2b
